src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -752,6 +752,8 @@ class HookUtils(object):
         """
         pbs.logmsg(pbs.EVENT_DEBUG4, "%s: Method called" % caller_name())
         # The cgroup was already deleted in the epilogue handler.
+        # Delete it again regardless, just in case
```

Comment:
How about: "The cgroup is usually deleted in the execjob_epilogue hook.  There are certain corner cases where that hook event will not be called.  We delete it again here to make sure we catch those.  It'll be a noop if the cgroup is already deleted"